assets/js/sidebar.js

```diff
@@ -0,0 +1,33 @@
+$(function() {
```

Comment:
Thanks for raising this @tinkertrain. I, too, like [reducing library use where possible](https://github.com/tagawa/bootstrap-without-jquery), but I think it requires deeper consideration here, for example the testing time it would require, or the increasing difference with the upstream Casper theme. I've started an internal task (just CCed you and @jagtalon on it) for us to consider the pros and cons.

Regarding this particular snippet, we'd have to do multi-browser testing first. As an example, jQuery gives us IE 9+ support whereas `classList` is only supported on IE 10+. We can always add more support with more edits but as we'd like to get these updates released soon, I think we should go with what we've currently got. However, I've pasted your snippet as a subtask in the internal discussion task so it won't be lost.